lib/App/dropboxapi.pm

```diff
@@ -49,7 +127,11 @@ dropbox-api help [<command>]
         put   upload file
         get   download file
         sync  sync directory (local => dropbox or dropbox => local)
-        uid   get accound uid
+
+    Common Options
+        -e enable env_proxy ( HTTP_PROXY, NO_PROXY )
+        -D enable debug
+        -v verbose
```

Comment:
Is -s documented here? It should be removed since it is no longer actually supported.